.codeclimate.yml

```diff
@@ -0,0 +1,36 @@
+engines:
+  rubocop:
+    enabled: true
+  scss-lint:
+    enabled: true
+  shellcheck:
+    enabled: true
+  eslint:
+    enabled: true
+  cofeelint:
```

Comment:
oh, yep

On Mon, May 23, 2016 at 1:01 PM pozorvlak <notifications@github.com> wrote:

> In .codeclimate.yml
> <https://github.com/Growstuff/growstuff/pull/879#discussion_r64251571>:
>
> > @@ -0,0 +1,36 @@
> > +engines:
> > +  rubocop:
> > +    enabled: true
> > +  scss-lint:
> > +    enabled: true
> > +  shellcheck:
> > +    enabled: true
> > +  eslint:
> > +    enabled: true
> > +  cofeelint:
>
> Should this be "coffeelint"?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/Growstuff/growstuff/pull/879/files/fd46fbbefe60ee4f03143a194ff2d2173f80240a#r64251571>
>